src/video/SDL_video.c

```diff
@@ -2225,7 +2225,7 @@ void SDL_SetWindowResizable(SDL_Window *window, SDL_bool resizable)
     CHECK_WINDOW_MAGIC(window, );
     if (!(window->flags & SDL_WINDOW_FULLSCREEN)) {
         const int want = (resizable != SDL_FALSE); /* normalize the flag. */
-        const int have = (window->flags & SDL_WINDOW_RESIZABLE);
+        const int have = ((window->flags & SDL_WINDOW_RESIZABLE) != 0);
```

Comment:
You can also do it like `!!(window->flags & SDL_WINDOW_RESIZABLE)`

Reply:
yes true, but the modification here was just a revert